Meetings/2022-03-09-meeting-minutes.md

```diff
@@ -0,0 +1,73 @@
+
+## Slides
+
+## Agenda
+
+- Review previous [meeting minutes](https://github.com/dotnet-foundation/wg-maintainers/blob/main/Meetings/2022-02-16-meeting-minutes.md)
+- Briefly discuss ()[https://github.com/dotnet-foundation/wg-maintainers/issues/48]
+- Talk about how we can provide better guidance to maintainers from this thread as well: [Centralised information for .NET Foundation operation](https://github.com/dotnet-foundation/Home/discussions/61)
+- Discuss #55 and #56
+- Other business
+
+## Notes
+
+- Reviewed previous meeting minutes
+- Process for leaving the foundation
+  - Board has voted to approve the process to leave the Foundation
+  - Tom Pappas met with the lawyers to review the process and to review the agreements that we will use going forward.
+  - We should put forward an FAQ for each agreement for joining the foundation
+  - Project committee will then review, then go to the Executive Committee an the Board
+  - Hoping to start July 1 with the new agreements and a clearer process for projects joining the foundation.
+- Want to have a joint meeting of the maintainers and project committees to help define our roles.
+- Project benefits PR, is being worked on and moving forward
+- Tom Pappas took the role of Executing Director Feb 1, will be announced in the next newsletter
+- Brief discussion about the probationary committee membership
+- Talk about how we can provide better guidance to maintainers from this thread as well: [Centralised information for .NET Foundation operation](https://github.com/dotnet-foundation/Home/discussions/61)
+  - We are starting to work on this from the board
+  - Tom Pappas wants to inventory the requests so that we can show sponsors what the Foundation does for projects
+  - [List of Topics)[https://github.com/dotnet-foundation/Home/discussions/61#discussioncomment-2085200] by @michael-hawker
+  - A very locatable location for this information would be good too. Eg, one problem with the DNF traditionally is just so hard to find any information
+  - Proposed a management repo to gather the information in a single location.
+  - Michael Hawker, A GitHub repo dedicated to a wiki could be a good implementation (and provide tracking for documenting individual questions/topics/processes). It could be auto-mirrored, so it could be centralized on the dotnet-foundation/Home repo as the 'Wiki' tab for instance. Example, we have our Wiki here on our main repo: [CommunityToolkit/WindowsCommunityToolkit Wiki](https://github.com/CommunityToolkit/WindowsCommunityToolkit/wiki) But it's auto-pushed and updated from this [dedicated repo](https://github.com/CommunityToolkit/WindowsCommunityToolkit-wiki).
+- DNF Account and CoC
+  - The CoC issue was a bug and unintentional
+  - Will reinstate all banned members
+  - Bans will be per org
+  - Email that was sent out announcing organization wide bans was confusing. Need to make it clearer and possibly add an FAQ.
+  - Ban list could be considered a shame list.
```

Comment:
I would add possibly has privacy laws implications if shared amongst all member projects that would need to be explored.